allure/allure.py

```diff
@@ -10,6 +11,8 @@ def get_time():
 
 class ReportBuilder(object):
     current_feature = ""
+    scenario_steps = []
```

Comment:
us I understand, this is private attribute, private attribute should begin with _